### DISCUSSION TITLE
Polarfire-soc-bare-metal-examples+video kit

### DISCUSSION THREAD

**Discussion by Balle0500** [polarfire-soc/polarfire-soc-documentation#393] [Bare metal embedded software]
If bare metal examples are compatible with the video kit? For example, DDR, LPDDR, Mac, etc.

**Comment by rockoguido** [comment]
there are a number of bare metal examples that have Video Kit support, see the latest version 2023.10.    all of the crypto examples are strictly for the Video Kit, there are 10 of those and the mfps-hal-ddr-demo also has been ported to the video kit

**Reply by Balle0500** [reply]
When I click on the ‘Details here’, there is no response. So how do I ensure that that these new paths are used in my project's include path?
<img width="1069" alt="image" src="https://github.com/polarfire-soc/polarfire-soc-documentation/assets/155931587/b29cc951-0085-4513-8f6d-b22864599d2e">
